browsers/samsunginternet_android.json

```diff
@@ -167,15 +167,21 @@
         },
         "12.0": {
           "release_date": "2020-06-19",
-          "status": "current",
+          "status": "retired",
           "engine": "Blink",
           "engine_version": "79"
         },
         "12.1": {
           "release_date": "2020-07-07",
-          "status": "current",
+          "status": "retired",
           "engine": "Blink",
           "engine_version": "79"
+        },
+        "13": {
```

Comment:
Wait, https://github.com/mdn/browser-compat-data/pull/6589 says "13.0", so this wasn't cherry-picked verbatim as the commit message would suggest.

Reply:
(I probably confused this with Safari versions when I was writing it... /)_-)